tasks/winbuildscripts/unittests.ps1

```diff
@@ -48,6 +48,12 @@ if($Env:TEST_WASHER){
     $TEST_WASHER_FLAG="--test-washer"
 }
 & inv -e install-tools
+& inv -e agent.build
+$err = $LASTEXITCODE
+if($err -ne 0){
+    Write-Host -ForegroundColor Red "building agent failed $err"
```

Comment:
Seems more natural like this:
```suggestion
    Write-Host -ForegroundColor Red "Agent build failed $err"
```

Reply:
I tried to stick to the current messages, I'll update them all